plio/views.py

```diff
@@ -183,17 +177,38 @@ def list_uuid(self, request):
 
         num_plios = queryset.count()
         queryset = self.filter_queryset(queryset)
-
-        uuid_list = queryset.values_list("uuid", flat=True)
-        page = self.paginate_queryset(uuid_list)
+        page = self.paginate_queryset(queryset.values())
 
         if page is not None:
+            # find the number of unique views
+            plio_ids = list(queryset.values_list(flat=True))
+
+            unique_user_counts = (
+                Session.objects.filter(plio__in=plio_ids)
+                .values("plio")
+                .annotate(count=Count("user", distinct=True))
+            ).order_by()
+
+            plio_id_to_count = {
+                unique_user_count["plio"]: unique_user_count["count"]
+                for unique_user_count in unique_user_counts
+            }
+
+            # add the number of unique viewers and items corresponding
+            # to the plio in each plio object
+            for index, plio in enumerate(page):
+                page[index]["num_views"] = plio_id_to_count.get(plio["id"], 0)
+                page[index]["items"] = ItemSerializer(
+                    queryset[index].item_set, many=True
+                ).data
```

Comment:
Why do we need items when i'm just doing `list` in the homepage? I just need top level plio details right?